app/core/services/ProcessPopulation.scala

```diff
@@ -103,4 +116,27 @@ trait ProcessPopulation {
           case Left(err) => Left(err)
         }
     }
+
+  private def processSequenceOptions(id: String, options: Seq[Phrase]): Either[GuidanceError,(Seq[Phrase], Boolean)] = {
+
+    val exclusiveOptions: Seq[Phrase] = options.filter{p => exclusiveOptionRegex.findAllMatchIn(p.english).nonEmpty}
+
+    exclusiveOptions.size match {
+      case size if size == 0 => Right((options, false))
+      case size if size == 1 =>
+        val nonExclusiveOptions: Seq[Phrase] = options.filter{p => exclusiveOptionRegex.findAllMatchIn(p.english).isEmpty}
+        nonExclusiveOptions.size match {
+          case nonExOpsSize if nonExOpsSize > 0 =>
+            val exclusiveOption: Phrase = Phrase(
+              exclusiveOptionRegex.replaceAllIn(exclusiveOptions.head.english,"").trim,
+              exclusiveOptionRegex.replaceAllIn(exclusiveOptions.head.welsh,"").trim
+            )
+            Right(((exclusiveOption +: nonExclusiveOptions.reverse).reverse, true))
+          case _ => Left(MissingNonExclusiveOptionError(id))
```

Comment:
if validation tightened, the MIssingNonExclusiveOptionError cannnot happen

Reply:
Fixed